.gitlab-ci.yml

```diff
@@ -13,11 +13,31 @@ stages:
     when: never
   - if: $CI_COMMIT_TAG != null
     when: never
+  - changes:
+    - "**/*.rb"
+    - Rakefile
+    - omnibus-software.gemspec
+    - Gemfile
+    when: always
+
+.on_success_skip_stable_branches: &on_success_skip_stable_branches
```

Comment:
I really think you should add a comment explaining why 2 anchors are needed, in order to avoid someone trying to simplify this in the future